qa/workunits/cephtool/test.sh

```diff
@@ -893,8 +893,13 @@ function test_mon_osd()
   # osd blacklist
   #
   bl=192.168.0.1:0/1000
+  # Escaped form which appears in JSON output
+  bl_json=192.168.0.1:0\\\\/1000
```

Comment:
bl_json='192.168.0.1:0\\/1000' would be easier to read if what you want is 192.168.0.1:0\\/1000 so that the double \\ is turned into a single \ by the grep command. At this stage of insanity I think using grep -F to not have regular expression escaping would make things a little less crazy and you could bl_json='192.168.0.1:0\/1000' to match 192.168.0.1:0\/1000 ;-)